cluster/ci/config/prow/plugins.yaml

```diff
@@ -150,6 +162,18 @@ plugins:
   openshift/console:
   - trigger
 
+  openshift/prometheus-operator:
+  - trigger
```

Comment:
Are these `trigger` statements necessary (seen in https://github.com/openshift/release/pull/859)?

Reply:
Yes, this is what tells prow to watch for changes to commits.

Reply:
This is what tells `prow` that you want observed changes to commits to trigger jobs. Prow sees the changes regardless.